packages/components/src/internal/components/permissions/InsufficientPermissionsAlert.tsx

```diff
@@ -17,4 +17,7 @@ import React, { FC } from 'react';
 
 import { Alert } from '../base/Alert';
 
-export const InsufficientPermissionsAlert: FC = () => <Alert>You do not have permissions for this action.</Alert>;
+export const InsufficientPermissionsAlert: FC<{ message?: string }> = ({ message }) => {
```

Comment:
Could also consider having `message` default to the value of `"You do not have permissions for this action."`

Reply:
Done